projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -61,12 +61,13 @@ export class AuthService {
       // that why we also need to check if we have access_token
       if (loginResult.result && token) {
         this.userIdService.setUserId(OCC_USER_ID_CURRENT);
-        this.store.dispatch(new AuthActions.Login());
 
         // We check if the token was received during the `tryLogin()` attempt.
         // If so, we will redirect as we can deduce we are returning from the authentication server.
         // Redirection should not be done in cases we get the token from storage (eg. refreshing the page).
+        // In this case we need to dispatch the login action to indicate that the user has just logged in.
         if (loginResult.tokenReceived) {
+          this.store.dispatch(new AuthActions.Login());
```

Comment:
`major:` I'm wondering if we can consider this change as a breaking, because we're dispatching in the different condition. Even if this is a fix, it might be safer to hide it under the feature flag, to avoid unexpected issues in client's application.